src/frontend/view/view.ts

```diff
@@ -70,30 +72,64 @@ const view = () => {
 }
 
 const viewSearchTypeSelector = () => `
-    <p class="govuk-body search-selector">
-      Search for:
-      <button class="${
-        state.searchParams.searchType === 'keyword' ? 'active' : ''
-      }" id="search-keyword">Keywords</button>
-      <button class="${
-        state.searchParams.searchType === 'link' ? 'active' : ''
-      }" id="search-link">Links</button>
-      <!-- Org search is disabled until we have tested a new design with users
-        <button class="${
-          state.searchParams.searchType === 'organisation' ? 'active' : ''
-        }" id="search-organisation">Organisations</button>
-      -->
-      <button class="${
-        state.searchParams.searchType === 'taxon' ? 'active' : ''
-      }" id="search-taxon">Taxons</button>
-      <button class="${
-        state.searchParams.searchType === 'language' ? 'active' : ''
-      }" id="search-language">Languages</button>
-      <button class="${
-        state.searchParams.searchType === 'advanced' ? 'active' : ''
-      }" id="search-advanced">Advanced</button>
-    </p>
-  `
+  <div class="govuk-tabs" data-module="govuk-tabs">
```

Comment:
can you add this after line 75   `<h2 class="govuk-tabs__title">Search for</h2>`